content/01.main_content.md

```diff
@@ -1,43 +1,45 @@
-  
-In award recognition, an audit of the International Society for Computational Biology (ISCB) honorees revealed significant under-representation of people with predicted East Asian name origins and over-representation of US-affiliated scientists.
-  After this study was publicized, ISCB's following honorees had the highest mean predicted probability of having an East Asian name of any previous year.
-  Additionally, the nominating committee inducted their first China-based fellow [@doi:10.1016/j.cels.2021.07.007].
+Auditing efforts can reduce disparities.
+  Journalists found that self-audits of gender-based representation in their articles can help them approach parity [@https://www.theatlantic.com/technology/archive/2016/02/gender-diversity-journalism/463023/; @https://www.theatlantic.com/science/archive/2018/02/i-spent-two-years-trying-to-fix-the-gender-imbalance-in-my-stories/552404/].
+  In award recognition, an audit of the International Society for Computational Biology (ISCB) honorees revealed significant under-representation of people with predicted East Asian name origins and over-representation of US-affiliated scientists.
+  After this study was publicized, ISCB's following honorees had the highest mean predicted probability of having an East Asian name of any previous year, and the nominating committee inducted their first China-based ISCB Fellow [@doi:10.1016/j.cels.2021.07.007].
 
 Audits can also provide evidence for specific actions.
   A recent study of disparities in scientific journalism found that predicted East Asian names were under-quoted and under-mentioned [@doi:10.1101/2021.06.21.449261].
+  <!--I think we want to be pretty careful here unless we're sure these particular results are statistically robust -->
   Interestingly, this disparity differed across the predicted name-origins of journalists.
   Furthermore, the differences between journalists was almost completely removed when considering people with a US-affiliated paper cited in the article.
   This suggests a regional bias, arguing for more regionally focused journalists.
+  <!--Maybe remove the above?-->
 
 Ideally, audits would only use self-reported gender, ethnicity, and other identifications [@doi:10.1371/journal.pcbi.1003903].
   While this is possible prospectively, surveying is impractical for large groups and often impossible retrospectively [@http://mediashift.org/2014/11/how-to-ethically-and-responsibly-identify-gender-in-large-datasets/].
-  In contrast, computationally-derived predictions allow for large-scale audits with as broad a scope as needed.
+  Computationally-derived predictions allow for large-scale audits with as broad a scope as needed.
```

Comment:
Was this taken out for the sake of the word count? I think it's doing useful work as a transition between ideas.

Reply:
Added back 👍